### PR TITLE
[SW-831] Check shape of H2O Frame after conversion from Spark frame

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

**Comment by h2o-ops** [comment]
Hi @jakubhava, your PR title "[PUBDEV-5340] Check shape of H2O Frame after conversion from Spark frame" is missing JIRA issue number! Please specify it in the following form [SW-]

File: core/src/main/scala/org/apache/spark/h2o/converters/WriteConverterCtxUtils.scala [hunk 1]
```diff
@@ -112,6 +112,22 @@ object WriteConverterCtxUtils {
           Log.debug(node + ": " + n + " chunks.")
       }
     }
+
+    val layouts = fr.vecs().map(_.espc())
+
+    if(!layouts.isEmpty){
```
Comment (michalkurka) [inline core/src/main/scala/org/apache/spark/h2o/converters/WriteConverterCtxUtils.scala]: What if ESPC is correct but the chunks itself are broken? Eg. there is a chunk that has less data than what the ESPC states. I think that might be the true cause of the bug

Comment (jakubhava) [reply core/src/main/scala/org/apache/spark/h2o/converters/WriteConverterCtxUtils.scala]: I see, good point. I will check also real shape of data

**Comment by jakubhava** [comment]
Added more checks

**Pull request merged**